db/schema.rb

```diff
@@ -182,6 +182,7 @@
     t.string "webpage"
     t.string "message"
     t.boolean "send_digest", default: false
+    t.boolean "send_close", default: false
```

Comment:
I think this might be a rebasing error. I usually like to run `rake db:migrate:reset` to make sure my `schema.rb` file is correct instead of just `rake db:migrate` or `rails db:schema:load` but if you don't want to mess with your db, I think removing this line should fix it

Reply:
Embarrassing! 😬 Must have rolled back one too few migrations after making the last demo branch. Thanks for catching https://github.com/SaraAlert/SaraAlert/pull/937/commits/187295c0d8009a6fd6eb5a2f21dbcd9e62052439